config/localTemplate.go

```diff
@@ -489,6 +489,11 @@ type Local struct {
 	// EnableExperimentalAPI enables experimental API endpoint. Note that these endpoints have no
 	// guarantees in terms of functionality or future support.
 	EnableExperimentalAPI bool `version[26]:"false"`
+
+	// NodeFollowerMode launches the node in "follower" mode. This turns off the agreement service,
+	// and APIs related to broadcasting transactions, and enables APIs which can retrieve detailed information
+	// from ledger caches and can control the ledger round.
+	NodeFollowerMode bool `version[27]:"false"`
```

Comment:
I kind of don't want it to be lumped in with all of the other "enable" settings just because it's also disabling certain functionality. But the other "Mode" settings are integers that serve as sort of enums whereas this is a boolean so I guess it does make sense to go in the "Enable" category just by virtue of its possible values.